lib/sigh/runner.rb

```diff
@@ -164,13 +165,12 @@ def download_profile(profile)
       profile_name ||= "#{profile.class.pretty_type}_#{Sigh.config[:app_identifier]}.mobileprovision" # default name
       profile_name += '.mobileprovision' unless profile_name.include? 'mobileprovision'
 
-      output_path = File.join('/tmp', profile_name)
-      File.open(output_path, "wb") do |f|
-        f.write(profile.download)
-      end
+      output = Tempfile.new(profile_name, mode: File::BINARY)
+      output.write(profile.download)
+      output.close
 
       Helper.log.info "Successfully downloaded provisioning profile...".green
-      return output_path
+      return profile_name, output
```

Comment:
How about we use the `Dir.mktmpdir("profile_download")` API instead? This way we get a temporary folder in which we can store the profile in its original name and don't need to return 2 values in this method?